java/client/src/main/java/glide/api/models/commands/WeightAggregateOptions.java

```diff
@@ -42,6 +43,23 @@ public String[] toArgs() {
         }
     }
 
+    /**
+     * Option for the method of aggregating scores from multiple sets. This option defaults to SUM if
+     * not specified.
+     */
+    public enum AggregateBinary {
+        /** Aggregates by summing the scores of each element across sets. */
+        SUM,
+        /** Aggregates by selecting the minimum score for each element across sets. */
+        MIN,
+        /** Aggregates by selecting the maximum score for each element across sets. */
+        MAX;
+
+        public GlideString[] toArgs() {
```

Comment:
Couldn't we use the non binary version here?
i.e. there is no need for binary data here... (toString() will return SUM, MIN or MAX and probably the constant `AGGREGATE_REDIS_API`) - seems a bit too much IMO